constants/deployments/deployments-g7-testnet.ts

```diff
@@ -189,4 +190,58 @@ export const G7_TESTNET_CONTRACTS: DeploymentContract[] = [
         functionCalls: [],
         args: StakerArgs.TESTNET,
     },
+    {
+        contractFileName: CONTRACT_UPGRADABLE_FILE_NAME.Chips,
+        type: CONTRACT_TYPE.Chips,
+        name: CONTRACT_NAME.Chips,
+        chain,
+        networkType,
+        tenants: [TENANT.Game7],
+        verify: true,
+        upgradable: true,
+        dependencies: [],
+        functionCalls: [
+            {
+                contractName: CONTRACT_NAME.Chips,
+                functionName: 'initialize',
+                args: [
+                    '0xD73EbC44643B594e44Ab46202590458c9606A709',
+                    false
+                ],
+            },
+        ],
+        args: [
+            '0xD73EbC44643B594e44Ab46202590458c9606A709',
```

Comment:
reuse a constant here to know in a few months this address what's without searching it in the explorer.